scripts/setup_hooks.py

```diff
@@ -74,7 +77,11 @@ def ensure_tool_installed(tool: str, force_update: bool = False) -> None:
     """
     if force_update or not which(tool):
         print(f"Ensuring latest {tool} via uv …")
-        run(["uv", "tool", "install", "--force", tool])
+        command = ["uv", "tool", "install", "--force", tool]
+        if python_ver:
+            # Add the Python version to the command if specified
+            command.extend(["--python", f"{python_ver[0]}.{python_ver[1]}"])
```

Comment:
nit: do you expect some people will not want you to install this? Should we ask the user to say yes before force installing stuff into their env here?